### PR TITLE
Plasma write_to_tex/dot unit tests

### PR DESCRIPTION
**Pull request by vg3095**

### CODE REVIEW

File: tardis/plasma/tests/test_plasma_dot_tex.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+data_path = os.path.join('tardis', 'plasma', 'tests', 'data')
+
+
+def test_write_dot(tmpdir, simulation_verysimple):
+    fname = str(tmpdir.mkdir('test_dot').join('plasma.dot'))
+    simulation_verysimple.plasma.write_to_dot(fname)
+    actual = pgv.AGraph(fname).to_string()
+    expected = pgv.AGraph(os.path.join(data_path, 'plasma_ref.dot')).to_string()
+    assert actual == expected
```
Comment (vg3095) [inline tardis/plasma/tests/test_plasma_dot_tex.py]: @wkerzendorf This passes on my Travis [build](https://travis-ci.org/vg3095/tardis/builds/267434622) for Ubuntu , not for Mac . And on Tardis Travis CI it fails for Ubuntu as well as for Mac. I can't understand the reason for this.

File: .travis.yml [hunk 2]
```diff
@@ -77,6 +77,8 @@ before_install:
     - if [[ $TEST_MODE == 'spectrum' ]]; then git lfs pull --include="atom_data/kurucz_cd23_chianti_H_He.h5" origin; fi
     - if [[ $TEST_MODE == 'spectrum' ]]; then git lfs pull --include="plasma_reference/" origin; fi
     - if [[ $TEST_MODE == 'spectrum' ]]; then cd $TRAVIS_BUILD_DIR; fi
+    - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then brew install graphviz; fi
```
Comment (wkerzendorf) [inline .travis.yml]: please use conda to install graphviz

File: tardis/plasma/base.py [hunk 3]
```diff
@@ -222,7 +222,7 @@ def write_to_dot(self, fname, latex_label=True):
                     print_graph.node[str(node)]['label']+=label
                     print_graph.node[str(node)]['label']+='$'
 
-        nx.write_dot(print_graph, fname)
+        nx.drawing.nx_agraph.write_dot(print_graph, fname)
```
Comment (wkerzendorf) [inline tardis/plasma/base.py]: thanks for changing that!

File: tardis/plasma/tests/test_plasma_dot_tex.py (hunk 1 above)
Comment (wkerzendorf) [inline tardis/plasma/tests/test_plasma_dot_tex.py]: well they are slightly different. can you post the diff of those files?

**Comment by wkerzendorf** [comment]
Travis complains about ` dot2tex missing. Plasma graph will not be generated.`

**Comment by vg3095** [comment]
@wkerzendorf It is working now.

**Pull request merged**